managers/forms.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, **kwargs: Any) -> None:
         super().__init__(
             queryset=TaskActivityType.objects.all(),
             required=False,
-            initial=TaskActivityType.objects.filter(is_default=True),
+            initial=TaskActivityType.objects.filter(is_active=True),
```

Comment:
You have method `get_active` which does exactly the same things as `filter(is_active=True)`. Please reuse it

Reply:
changed